okhttp/src/main/java/okhttp3/internal/platform/Platform.java

```diff
@@ -183,6 +183,15 @@ public CertificateChainCleaner buildCertificateChainCleaner(SSLSocketFactory ssl
 
   /** Attempt to match the host runtime to a capable Platform implementation. */
   private static Platform findPlatform() {
+    if ("conscrypt".equals(System.getProperty("okhttp.platform"))) {
```

Comment:
It might be better to just check if Conscrypt is the first installed Provider?

Reply:
oooh I like that option. Then we’re relying on the same mechanism the Java security infrastructure wants us to. I like that. Can be now or in follow-up.